dev/src/configs.clj

```diff
@@ -0,0 +1,65 @@
+(ns configs
+  (:require [clojure.java.io :as io]
+            [aero.core :as aero]
+            [clojure.string :as str]
+            [meta-merge.core :refer [meta-merge]]
+            [integrant.core :as ig]))
+
+(defn load-config
+  "Loads aero config at given file path using profile keyword."
+  [config-file profile]
+  (-> config-file
+      io/resource
+      (aero/read-config {:profile profile})))
+
+(defn dev-config
+  []
+  (load-config "config.edn" :dev))
+
+;; raft configs
+
+(def server-1 "/ip4/127.0.0.1/tcp/62071")
+(def server-2 "/ip4/127.0.0.1/tcp/62072")
+(def server-3 "/ip4/127.0.0.1/tcp/62073")
+(def servers-str (str/join "," [server-1 server-2 server-3]))
+
+(def raft-single-server
```

Comment:
I think this stuff could be stored in resource files on the classpath, just like we do the main config. We could add a development resource path under `dev/resources` and call those files `dev-server-1.edn` (and hopefully very soon ".json"), etc and build out functions to merge or override configs in the `fluree.server.system` (and, hopefully very soon, the `fluree.server.config`) namespace. That would make it more robust and also more like what we'll do in production.